com.unity.shadergraph/Editor/Data/Graphs/Vector2MaterialSlot.cs

```diff
@@ -20,6 +20,9 @@ class Vector2MaterialSlot : MaterialSlot, IMaterialSlotHasValue<Vector2>
         [SerializeField]
         string[] m_Labels; // this can be null, which means fallback to k_LabelDefaults
 
+        [SerializeField]
```

Comment:
I think we can get by without serializing this.  The slots are reinitialized after deserialization, so it will get stomped anyways...

(the fact that we serialize the slots directly instead of just the default values is kind of silly...  going to fix that with the GTF refactor)

Reply:
Yes you are right, this is always setup by the slot, that doesn't need to be serialized, I will remove that.